components/header/_header.scss

```
@@ -14,11 +14,15 @@
     }
   }
 
-  // Height of crown is 30px.
-  // Visually adjust placement to accept larger icons with a height of 35px
-  .app-header__logotype--large {
-    margin-bottom: -10px;
-    position: relative;
-    top: -3px;
+  // Visually adjust placement of X-GOVUK logo which has a height of 35px
+  .govuk-header__logotype[aria-label="X-GOVUK"] {
```

Could use a separate class name here?

```suggestion
  .x-govuk-header__logotype {
```